include/odp/api/spec/event.h

```diff
@@ -209,6 +209,44 @@ void odp_event_free_multi(const odp_event_t event[], int num);
  */
 void odp_event_free_sp(const odp_event_t event[], int num);
 
+/**
+ * Event flow id value
+ *
+ * Returns the flow id value set in the event.
+ * Usage of flow id enables scheduler to maintain multiple synchronization
+ * contexts per single queue. For example, when multiple flows are assigned to
+ * an atomic queue, events of a single flow (events from the same queue with
+ * the same flow id value) are guaranteed to be processed by only single thread
+ * at a time.  For packets received through packet input initial
+ * event flow id will be same as flow hash generated for packets. The hash
+ * algorithm and therefore the resulting flow id value is implementation
+ * specific. Use pktio API configuration options to select the fields used for
+ * initial flow id calculation. For all other events initial flow id is zero
+ * An application can change event flow id using odp_event_flow_id_set().
+ *
+ * @param	event	Event handle
+ *
+ * @return		Flow id of the event
+ *
+ */
+uint32_t odp_event_flow_id(odp_event_t event);
```

Comment:
This commit cannot move yet since odp_schedule_config_t is not moved: "api: schedule: add scheduler flow aware mode" (791450618ce075cd10130f953e0f895324681cf1)